modules/packages/LinearAlgebra.chpl

```diff
@@ -1267,28 +1272,28 @@ private proc _lu (in A: [?Adom] ?eltType) {
       A[i,..] <=> A[swaprow,..];
       L[i,..] <=> L[swaprow,..];
       ipiv[i] <=> ipiv[swaprow];
-      numSwap+=1;
+      numSwap += 1;
     }
 
-    forall k in i..n {
-      var sum = + reduce (L[i,..] * U[..,k]);
+    forall k in i..<n {
+      const sum = + reduce (L[i,..] * U[..,k]);
```

Comment:
Why was this sum variable changed from a var to a const?

Reply:
The variable is not intended to change elsewhere. When this is the case, I prefer to put a `const` there as a way to make the code more self-documenting and to lock in that behavior.

Reply:
Understood.  Thank you.